cddl/verificationMethodTypes.cddl

```diff
@@ -9,5 +9,6 @@ VerificationMethodTypes =
     "GpgVerificationKey2020" / 
     "RsaVerificationKey2018" / 
     "X25519KeyAgreementKey2019" /
-    "EcdsaSecp256k1RecoveryMethod2020"
+    "EcdsaSecp256k1RecoveryMethod2020" /
```

Comment:
@msporny @iherman @burnburn  this is why leaving these files in this repo is problematic... folks are doing more work than is required, but I am not actually doing to review the CDDL, because the WG agreed it was not required.

Reply:
Agree that CDDL should be removed as soon as possible.